docs/reST/ref/sdl2_video.rst

```diff
@@ -183,6 +183,16 @@
       | :sl:`Copy a portion of the texture to the rendering target.`
       | :sg:`draw(srcrect=None, dstrect=None, angle=0, origin=None, flip_x=False, flip_y=False) -> None`
 
+   .. method:: draw_triangle
+      
+      | :sl:`Copy a triangle portion of the texture to the rendering target by vertices coordinates.`
+      | :sg:`draw_triangle(p1_xy, p2_xy, p3_xy, p1_uv=(0.0, 0.0), p2_uv=(1.0, 1.0), p3_uv=(0.0, 1.0), p1_mod=(255, 255, 255, 255), p2_mod=(255, 255, 255, 255), p3_mod=(255, 255, 255, 255)) -> None`
+
+   .. method:: draw_quad
+      
+      | :sl:`Copy a quad portion of the texture to the rendering target by vertices coordinates.`
+      | :sg:`draw_quad(p1_xy, p2_xy, p3_xy, p4_xy, p1_uv=(0.0, 0.0), p2_uv=(1.0, 0.0), p3_uv=(1.0, 1.0), p4_uv=(0.0, 1.0), p1_mod=(255, 255, 255, 255), p2_mod=(255, 255, 255, 255), p3_mod=(255, 255, 255, 255), p4_mod=(255, 255, 255, 255)) -> None`
```

Comment:
```suggestion
      | :sg:`draw_quad(xy, uv=((0.0, 0.0), (1.0, 0.0), (1.0, 1.0),(0.0, 1.0)), mod=((255, 255, 255, 255),) * 4) -> None`
```
The function signatures here could look something like this. The `draw_triangle` function could also be updated similarly

Reply:
Anyway, I'm back after 6 months. I don't think I agree with this change. `pygame.draw.line` takes seperate `start_pos` and `end_pos` arguments; and triangles and quads have exact amounts of points anyway, unlike something like `pygame.draw.polygon`.